### PR TITLE
return error information from CapturePanic

### PR DESCRIPTION
**Pull request by codekitchen**
We use this to log the sentry error ID and return it in our API response.

### CODE REVIEW

**Comment by mattrobenolt** [comment]
TIL that you could do this from within a `defer`. I guess I have never thought about it before. :) https://play.golang.org/p/jSjucJaUcG

File: client.go [hunk 1]
```diff
@@ -555,14 +555,16 @@ func CaptureErrorAndWait(err error, tags map[string]string, interfaces ...Interf
 }
 
 // CapturePanic calls f and then recovers and reports a panic to the Sentry server if it occurs.
-func (client *Client) CapturePanic(f func(), tags map[string]string, interfaces ...Interface) {
+// If an error is captured, both the error and the reported Sentry error ID are returned.
+func (client *Client) CapturePanic(f func(), tags map[string]string, interfaces ...Interface) (err interface{}, errorID string) {
```
Comment (mattrobenolt) [inline client.go]: Couldn't `err` be an `error` here?

Comment (mattrobenolt) [inline client.go]: Nevermind, I forgot this is what `recover()` returns: https://golang.org/pkg/builtin/#recover

**Comment by mattrobenolt** [comment]
:cake: :icecream: 

Thanks!

**Comment by codekitchen** [comment]
Ha yeah I was stuck for a second until I thought to try named return vals. Thanks!

**Pull request merged**